### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
- Updates OTP-UI dependencies
- Updates `create-OTP-reducer` to avoid crash

**PR Checklist:**
- [n/a] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [n/a] Are all languages supported (Internationalization/Localization)?
- [n/a] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by daniel-heppner-ibigroup — APPROVED** [review]
LGTM, thanks for the updates!

File: lib/components/narrative/metro/default-route-renderer.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const DefaultRouteRenderer = ({
   leg,
   style
 }: RouteRendererProps): JSX.Element => {
-  const routeTitle = leg.routeShortName || leg.route || leg.routeLongName
+  const routeTitle = leg.routeShortName || leg.routeLongName
```
Comment (miles-grant-ibigroup) [inline lib/components/narrative/metro/default-route-renderer.tsx:37]: I think `leg.route` is sometimes a value. Can we do `leg?.route` instead?

Comment (miles-grant-ibigroup) [reply lib/components/narrative/metro/default-route-renderer.tsx:37]: and while we're at it `leg?.routeShortName` and `leg?.routeShortName`

File: package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "@bugsnag/plugin-react": "^7.17.0",
     "@floating-ui/react": "^0.19.2",
     "@opentripplanner/base-map": "^3.0.13",
-    "@opentripplanner/core-utils": "^9.0.3",
+    "@opentripplanner/core-utils": "^11.0.0",
```
Comment (miles-grant-ibigroup) [inline package.json:42]: Once we're upgrading can we please also update `itinerary-body`? `yarn upgrade -S @opentripplanner --latest` should give us all we need

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Just a few comments!

**Comment by miles-grant-ibigroup** [comment]
Also snapshots need to be updated!

**Pull request merged**